### PR TITLE
Shockwave

### PR DESCRIPTION
**Pull request by TheSilverHornet**
Note that https://github.com/beyond-all-reason/Beyond-All-Reason/pull/2145 and https://github.com/beyond-all-reason/BYAR-Chobby/pull/486 are prerequisites, please do not merge until those are in place

### CODE REVIEW

File: units/ArmBuildings/LandDefenceOffence/armshockwave.lua [hunk 1]
```diff
@@ -28,13 +29,16 @@ return {
...
+		mincloakdistance = 50,		
 		nochasecategory = "MOBILE",
 		objectname = "Units/armshockwave.s3o",
 		onoffable = false,
 		script = "Units/armshockwave.cob",
 		seismicsignature = 0,
 		selfdestructas = "mediumBuildingExplosionGeneric",
 		sightdistance = 455,
+		sonarstealth = true,
+		stealth = true,
```
Comment (drivver44) [inline units/ArmBuildings/LandDefenceOffence/armshockwave.lua:41]: add
		usepiececollisionvolumes = 1,

**Review by drivver44 — CHANGES_REQUESTED** [review]
addition to collisionvolumes.lua
```
pieceCollisionVolume['armshockwave'] = {
	['2']={22,22,22,0,10,0,1,1},
	['0']={60,65,60,0,20,0,1,1,0},
}
```

File: units/other/raptors/Kamikaze/raptor_dodo1.lua [hunk 2]
```diff
@@ -39,7 +39,7 @@ return {
 		movementclass = "RAPTORSMALLHOVER",
 		noautofire = false,
 		nochasecategory = "VTOL HOVER",
-		objectname = "Raptors/raptor_dodo.s3o",
+		objectname = "Raptors/boomertailtoo.s3o",
```
[the remark this answers is not in the document]
Comment (TheSilverHornet) [reply units/other/raptors/Kamikaze/raptor_dodo1.lua:42]: This is a draft for a reason, don't worry, it's only here so drivver could easily grab things he needed from it

File: language/en/units.json [hunk 3]
```diff
@@ -199,6 +199,7 @@
 			"armsh": "Seeker",
 			"armshltx": "Experimental Gantry",
 			"armshltxuw": "Experimental Gantry",
+			"armshockwave": "Advanced Defensive Metal Extractor",
```
[the remark this answers is not in the document]
Comment (TheSilverHornet) [reply language/en/units.json:202]: It's called Shockwave, this is old and somewhat placeholder

**Pull request merged**